admin/acceptance.py

```diff
@@ -305,6 +343,8 @@ def main(args, base_path, top_level):
 
     runner = options.runner
 
+    signal.signal(signal.SIGTERM, signal_handler)
```

Comment:
I added some. I could add some more, referring to the C implementation, but I think what I added is probably sufficient.